### DISCUSSION TITLE
MediaRecorder doesn't work properly in Safari

### DISCUSSION THREAD

**Discussion by bbbbbgdn** [cables-gl/cables_docs#640] [! Welcome - readme first!]
In Safari (Version 17.1 (19616.2.9.11.7)) there's a bug with default MediaRecorder behaviour:
It just doesn't get's any proper mime type at the start.

this example: https://cables.gl/p/ioiDIR
<img width="1101" alt="Screenshot 2023-12-18 at 21 01 32" src="https://github.com/cables-gl/cables_docs/assets/12091461/9c41391e-1db9-4d27-8bcb-d113af46afc2">

Here is workaround that works in Safari, it seems that Mime type needs to be set with slight delay
https://cables.gl/p/y7txQK

**Comment by pandrr** [comment]
moved to: https://github.com/cables-gl/cables_docs/issues/new?created_from_discussion_number=640
